doc/misc/npm-registry.md

```diff
@@ -21,6 +21,21 @@ The registry URL used is determined by the scope of the package (see
 supplied by the `registry` config parameter.  See `npm-config(1)`,
 `npmrc(5)`, and `npm-config(7)` for more on managing npm's configuration.
 
+## Does npm send any information about me back to the registry?
```

Comment:
i'd also mention if the data sent is tied to username or package. i.e. is it "anonymized", not taken into account with any other data. also is the scope and the ci data going to be analyzed together or kept separate. people might worry about rate limiting.